Libraries/LibDNS/Resolver.h

```diff
@@ -40,7 +72,8 @@ class LookupResult : public AtomicRefCounted<LookupResult>
             re.record.record.visit(
                 [&](Messages::Records::A const& a) { result.append(a.address); },
                 [&](Messages::Records::AAAA const& aaaa) { result.append(aaaa.address); },
-                [](auto&) {});
+                [](auto&) {
+                });
```

Comment:
some of the clang-format changes in this file are sus

Reply:
:yakshrug: clang-format 20.1.0 doesn't change anything when run, guess I'll manually reformat and see what it does.